src/steps/mdx-to-gridtables.ts

```diff
@@ -125,7 +111,9 @@ export default function mdxToBlocks(ctx: Helix.UniversalContext) {
       ],
     }));
 
-    const totalSlots = repeat * slots.length;
+    // Determine how many children to treat as slots
+    const totalSlots = slotsValue === 'none' ? 0 : repeat * slots.length;
```

Comment:
```suggestion
    const totalSlots = blockName === 'HorizontalLine' ? 0 : repeat * slots.length;
```

then can delete lines 74-77, and 81-84